src/open_inwoner/openzaak/models.py

```diff
@@ -563,11 +564,11 @@ class Meta:
             )
         ]
 
-    def has_received_similar_notes_within(self, period: timedelta) -> bool:
+    def has_received_similar_notes_within(
+        self, period: timedelta, template_name: str
+    ) -> bool:
         return UserCaseStatusNotification.objects.has_received_similar_notes_within(
-            self.user, self.case_uuid, period, not_record_id=self.id
-        ) or UserCaseInfoObjectNotification.objects.has_received_similar_notes_within(
-            self.user, self.case_uuid, period
```

Comment:
I wasn't too sure about removing this, but in practice we never send `UserCaseStatusNotification`s that have the same `template_name` as `UserCaseInfoObjectNotification`s

Reply:
"in practice.. (at this point in time)", famous last words. It's is probably fine but an assumption and not a hard enforced rule. Let's see what happens.